src/browser/index.js

```diff
@@ -42,17 +43,40 @@ if (isReload) {
 document.body.classList.toggle('use-native-titlebar',
                                Runtime.useNativeTitlebar());
 
+const replay =
+  () => {
+    const request = new XMLHttpRequest();
+    // We use sync XHR as we have nothing else to render otherwise.
```

Comment:
> I'd prefer. It's just about getting the same behavior with gecko and servo. But if it's too much work, just forget about it.

This actually does not work on gecko, because we can't pass query arguments there, we use url to manifest instead :) Honestly I want to change an interaction quite a bit which is why I did not wanted to bother with this too much.